dockstore-common/src/main/java/io/dockstore/common/yaml/constraints/HasEntryValidator.java

```diff
@@ -13,17 +13,18 @@
  *    See the License for the specific language governing permissions and
  *    limitations under the License.
  */
-package io.dockstore.common.yaml;
+package io.dockstore.common.yaml.constraints;
 
+import io.dockstore.common.yaml.DockstoreYaml12;
 import javax.validation.ConstraintValidator;
 import javax.validation.ConstraintValidatorContext;
 
 /**
  * Validates that a DockstoreYaml12 instance has at least one workflow, tool, or service
```

Comment:
Is no longer clear whether this is for Dockstore yaml schema 1.2 or is intended to be futureproof
(Generic type seems to imply 1.2 but name is versionless)